cloudx/cmd_auth_test.go

```diff
@@ -70,6 +70,17 @@ func TestAuthenticator(t *testing.T) {
 			expectSignInSuccess(t)
 		})
 
+		t.Run("forced to reauthenticate on session expiration", func(t *testing.T) {
+			cmd := configAwareCmd(configDir)
+			expectSignInSuccess(t)
+			changeAccessToken(t, configDir)
+			var r bytes.Buffer
+			r.WriteString("n\n") // Your CLI session has expired. Do you wish to login again as <email>?
+			_, stderr, err := cmd.ExecDebug(t, &r, "list", "projects")
+			require.Error(t, err)
+			assert.Contains(t, stderr, "Your CLI session has expired. Do you wish to log in again as")
```

Comment:
Can you please add the log in sequence here (see the other tests), to ensure that we are actually able to log in properly?